### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tandelDipak**
Added life expectancy column. It will display NaN for a country that is not matched or not available in API.

---
This change is [Reviewable](https://reviewable.io/reviews/serhii73/place2live/139)

### CODE REVIEW

**Comment by tandelDipak** [comment]
Hi @serhii73 
I am sending another pull request with minor changes pass checks.

**Comment by serhii73** [comment]
Thank you very much for your PR.
Can you please ti fix the build errors? https://github.com/serhii73/place2live/actions

**Comment by tandelDipak** [comment]
@serhii73  Do I have to send another PR with changes?

**Comment by tandelDipak** [comment]
Hi @serhii73 
Can you help with the reorder python import? The build is getting failed again and again because of that. I don't know the reason.

**Comment by serhii73** [comment]
It is checked match code PEP8 or not.
You can try:
```
pip install isort
isort where.py
```

**Comment by serhii73** [comment]
Or
```bash
pip install reorder-python-imports
reorder-python-imports where.py
```

**Comment by serhii73** [comment]
```python
from datetime import datetime

import pandas as pd
import requests
from utils import text_color
from utils import text_type
```

**Comment by serhii73** [comment]
> ```python
> from datetime import datetime
> 
> import pandas as pd
> import requests
> from utils import text_color
> from utils import text_type
> ```

@tandelDipak try it. 
Thank you very much!

**Comment by tandelDipak** [comment]
@serhii73 can you give me little info about these pipelines. How it is built etc.

**Comment by serhii73** [comment]
[My pre-commit config](https://github.com/serhii73/place2live/blob/master/.pre-commit-config.yaml)
[Read more about pre-commit](https://pre-commit.com/)
[The config for github actions](https://github.com/serhii73/place2live/blob/master/.github/workflows/pythonapp.yml)

File: where.py [hunk 1]
```diff
@@ -34,6 +37,65 @@ def run_country_checker():
...
+YOUR_AGE = run_age_checker()
+
+
+def run_gender_checker():
+    """Take valid gender from user"""
+    while True:
+        try:
+            gender = input(
+                text_color(
+                    "What is their gender Male or Female? ", text_type.QUESTION,
```
Comment (serhii73) [inline where.py]: Why `their`?

File: where.py [hunk 2]
```diff
@@ -34,6 +37,65 @@ def run_country_checker():
...
+
+YOUR_GENDER = run_gender_checker()
+
+
+def get_url(country, age, gender):
+    """Construct URL based on input"""
+    country = country.title()
+    date = datetime.today().strftime('%Y-%m-%d')
+    url_part1 = "https://d6wn6bmjj722w.cloudfront.net/1.0/life-expectancy/remaining/"
+    url_part2 = gender + "/" + country + "/" + date + "/" + age + "y/?format=json"
```
Comment (serhii73) [inline where.py]: Why not `format` or `f` strings?

Comment (tandelDipak) [reply where.py]: It's updated

**Comment by tandelDipak** [comment]
@serhii73 is everything fine now or still, there is some issue?

**Comment by serhii73** [comment]
Sorry, I will review it on Monday. Is it ok?
Thank you very much for your PR.

**Comment by tandelDipak** [comment]
Ya, it's fine.

**Pull request merged**